src/Item/Item.service.ts

```diff
@@ -645,4 +626,25 @@ export class ItemService {
     }
     return Bridge.toFullItem(upsertedItem, dbCollection)
   }
+
+  private async checkIfThirdPartyItemURNExists(
+    id: string,
+    urn: string,
+    action = ItemAction.UPSERT
+  ): Promise<void> {
+    const decodedItemURN = decodeThirdPartyItemURN(urn)
+    if (
+      await Item.isURNRepeated(
+        id,
+        decodedItemURN.third_party_id,
+        decodedItemURN.item_urn_suffix
+      )
+    ) {
+      throw new URNAlreadyInUseError(id, urn, action)
+    }
+    const [wearable] = await peerAPI.fetchWearables<ThirdPartyWearable>([urn])
+    if (wearable) {
+      throw new URNAlreadyInUseError(id, urn, ItemAction.INSERT)
```

Comment:
Why is it using `ItemAction.INSERT` here?

Reply:
Nope, just a leftover! thanks!